src/src/utils/getConfig.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import { getToken, getOrganizationId, getUsername, getUserId } from '@/utils/auth';
+import { getTenant, solutionId } from '@/extension/extra';
+
+export function getConfig(item) {
+    let envUrl;
+    let instance;
+    if (Object.keys(config).length > 0) {
+        envUrl = config[item];
+        instance = axios.create({  baseURL: envUrl });
+    } else {
+        envUrl = process.env[item];
+        instance = axios.create({ baseURL: envUrl })
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
    instance = axios.create({ baseURL: envUrl })
```